### PR TITLE
drop backward compatibility with GLPI < 9.1.2

### PR DESCRIPTION
**Pull request by btry**

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -159,6 +159,10 @@ Features
          <compatibility>0.90</compatibility>
          <compatibility>9.1</compatibility>
       </version>
+      <version>
```
Comment (orthagh) [inline plugin.xml]: don't publish the version yet, we didn't release it

**Comment by btry** [comment]
PR rebased

**Comment by orthagh** [comment]
but tests fail :)

**Comment by orthagh** [comment]
btw, my review is still valid

**Comment by btry** [comment]
PR rebased due to code refactor for target ticket and target change #586.

Target change contains code for backward compatibility to be removed here

**Comment by btry** [comment]
needs new validation du to big rebase + drop of code in targetchange

**Pull request merged**